### PR TITLE
Bugfix/#297

### PR DESCRIPTION
**Pull request by szrtGreg**
Resources messages #297 

## Description
Created two resources messages for Views->Test->Index

## Motivation and context
#297 

## Types of changes

- [ x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by szrtGreg** [comment]
In the mentioned file (Test->Index.cshtml) was more plain text but I was not sure if I need to change it. Example:

"<div class="row content-centered">
                            <div class="col text-center">
                                <h4>Gdy na zegarze 20 wybije</h4>
                                <h4>okienko swoją zagadkę odkryje...</h4>
                                <h4>Wiec spiesz się i co dzień okienka otwieraj.</h4>
                                <h4>Zdobywaj punkty i nagrody zbieraj!</h4>
                            </div>
                        </div>"

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml [hunk 1]
```diff
@@ -87,7 +88,7 @@
                         }
                         else
                         {
-                            <h3 class="mb-30 text-primary">Niestety, spóźniłeś się z odpowiedzią na to pytanie :(</h3>
+                            <h3 class="mb-30 text-primary">@DelayedAnswer</h3>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml:91]: `ItsTooLateToAnswer` or `YouAreLateWithAnswer` should be better here

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml [hunk 2]
```diff
@@ -68,7 +69,7 @@
 
                         if (Model.HasUserAnswered)
                         {
-                            <h3 class="mb-30 text-primary">Ho ho! Już odpowiedziałeś na to pytanie, brawo :)</h3>
+                            <h3 class="mb-30 text-primary">@HasUserAnsweredQuestion</h3>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Test/Index.cshtml:72]: `UserAlreadyAnswered` should be better here

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsTestMessages.resx [hunk 3]
```diff
@@ -0,0 +1,126 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Resources/ViewsTestMessages.resx:1]: we could create one .resx file for all resources in all views - so maybe we should consider renaming it to `ViewsMessages`?

**Comment by WTobor** [comment]
> In the mentioned file (Test->Index.cshtml) was more plain text but I was not sure if I need to change it. 
> "

We should check it - debug :) we can discuss it later today

**Pull request closed without merging**